task_1/scss/sliderSection.scss

```diff
@@ -0,0 +1,214 @@
+@import './_variables';
```

Comment:
you don't have to import variables in every file. You don't need separate css files. Your gulp should process only main.scss where you will import all files.